01-Introduction.qmd

```diff
@@ -104,5 +104,5 @@ In [Chapter 7](07-glossary.html) a glossary of statistical terms used in Statist
 Duncan, G., Keller-McNulty, S., and Stokes, S. (2001) *Disclosure Risk vs. Data Utility: the R-U Confidentiality Map,* Technical Report LA-UR-01-6428, Statistical Sciences Group, Los Alamos, N.M.: Los Alamos National Laboratory
 
 Trewin, D et al. (2007) *Principles and Guidelines of Good Practice for Managing Statistical Confidentiality and Microdata Access*, UNECE United Nations Economic commission for Europe: 
-[[https://unece.org/.../Managing.statistical.confidentiality.and.microdata.access.pdf]{.underline}](https://unece.org/fileadmin/DAM/stats/publications/Managing.statistical.confidentiality.and.microdata.access.pdf)
+[[https://unece.org/fileadmin/DAM/stats/publications/Managing.statistical.confidentiality.and.microdata.access.pdf]{.underline}](https://unece.org/fileadmin/DAM/stats/publications/Managing.statistical.confidentiality.and.microdata.access.pdf)
```

Comment:
I added ... dots in the visible url because in the pdf file the url can't be cut smartly to avoid that the URL is written in the margins of the pdf. Indeed, a URL is smartly cut (back to the line) in general but here it doesn't work because of the dots in the url: "Managing.statistical.confidentiality.and.microdata.access.pdf" is not automatically cut. Except if you find a way to avoid this misbehaviour, I would choose to restore the ... dots. 
Anyway, people can have the real url by clicking on it or by copy-pasting it with a right-click.

Reply:
Okay, I understand. 

I guess that then only printed pdf's have a "problem" (they do not show the complete link).

The only simple solution I can find is removing the {.underline} (I added it because in the pdf you would like the url's to be shown as links, e.g., underlined, and that does not happen automaticly).
If that's a convenient solution to you, let me know and I'll remove all {.underline}'s at the appropriate places (and leave them where they are needed for other reasons).

Reply:
I've tested your proposition to remove the .underline property. And it's seems to work. 
Without .underline, the link is blue. Maybe we could remove all the .underline.

If you lack some time, I can do it tomorrow morning.

Reply:
I'll check tomorrow morning. I have some time then.